src/ui/components/Header/UserOptions.js

```diff
@@ -36,6 +36,14 @@ function UserOptions({ recordingId, setModal }) {
     }
     window.location = dashboardUrl;
   };
+  const onDocsClick = () => {
+    const docsUrl = `https://www.notion.so/Docs-56758667f53a4d51b7c6fc7a641adb02`;
+
+    if (event.metaKey) {
+      return window.open(docsUrl, "_blank");
+    }
+    window.open("https://www.notion.so/Docs-56758667f53a4d51b7c6fc7a641adb02", "_blank");
```

Comment:
why not re-use the same variable?

Also, shouldn't this one use something other than blank?

Reply:
Apologies! This is an old branch. See here: https://github.com/RecordReplay/devtools/pull/2818/files